### PR TITLE
Fix circular dependency import at fromMovingPoints file

### PR DESCRIPTION
**Pull request by jedrzejchalubek**
Rollup reporting circular dependency import at `fromMovingPoints` file

```
Circular dependency: ../../node_modules/transformation-matrix/src/index.js -> ../../node_modules/transformation-matrix/src/fromMovingPoints.js -> ../../node_modules/transformation-matrix/src/index.js
```

### CODE REVIEW

File: src/fromMovingPoints.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { translate } from './translate'
 import { applyToPoint } from './applyToPoint'
-import { scale, rotate, compose } from './index'
+export { rotate } from './rotate'
+export { scale } from './scale'
+export { compose } from './transform'
```
Comment (chrvadala) [inline src/fromMovingPoints.js:5]: I understand this issue, but I think that the right word here is `import` rather then `export`

**Review by chrvadala — CHANGES_REQUESTED** [review]

**Comment by chrvadala** [comment]
Thanks @jedrzejchalubek  for your useful PR,
I added a comment on it, by the way, I suggest you to run `npm test` before committing. It can save your time

File: src/fromMovingPoints.js (hunk 1 above)
Comment (jedrzejchalubek) [reply src/fromMovingPoints.js]: Ah, right 🙃 Fixed!

**Comment by chrvadala** [comment]
Released with v2.15

**Pull request merged**